.github/workflows/pull_request.yml

```diff
@@ -50,9 +46,7 @@ jobs:
         run: |
           make versions
           
-          echo "matrix=$( cat versions/strategyMatrix.json )" >> "$GITHUB_OUTPUT"
-          echo "linux_matrix=$( cat versions/linuxMatrix.json )" >> "$GITHUB_OUTPUT"
-          echo "windows_matrix=$( cat versions/windowsMatrix.json )" >> "$GITHUB_OUTPUT"
+          echo "linux_and_windows_matrix=$( cat versions/linuxAndWindowsMatrix.json )" >> "$GITHUB_OUTPUT"
```

Comment:
Cleaning this up a little bit: only the matrix containing windows and linux (non-suse) distros are used from the PR workflow for now.